libs/coin-evm/src/__tests__/unit/synchronization.unit.test.ts

```diff
@@ -523,12 +525,32 @@ describe("EVM Family", () => {
             } as any),
         );
 
-        expect(await synchronization.getOperationStatus(currency, coinOperations[0])).toEqual({
-          ...coinOperations[0],
+        const expectedAddition = {
           blockHash: "hash",
           blockHeight: 10,
           date: new Date(),
           transactionSequenceNumber: 123,
+        };
+
+        expect(
+          await synchronization.getOperationStatus(currency, {
+            ...coinOperations[0],
+            subOperations: [tokenOperations[0]],
+            nftOperations: [erc721Operations[0], erc1155Operations[0]],
+          }),
+        ).toEqual({
```

Comment:
```suggestion
        const operationStatus =  await synchronization.getOperationStatus(currency, {
            ...coinOperations[0],
            subOperations: [tokenOperations[0]],
            nftOperations: [erc721Operations[0], erc1155Operations[0]],
          });
        expect(operationStatus).toEqual({
```

not mandatory but I think it's clearer if we don't have an await inside a function

Reply:
Updated 👍